content/docs/services/cdn-route.md

```diff
@@ -209,6 +213,12 @@ Other headers, such as HTTP auth, are stripped by default.
 
 If you need a different configuration, contact [cloud.gov support](/help/).
 
+### DNSSEC Support
+
+Custom domains using the CDN broker will not fully validate DNSSEC (between your CNAME record and cloudfront.net). This is because the AWS CloudFront service does not currently support DNSSEC. For a complete implementation of DNSSEC, we recommend instead running a proxy server within your boundary that forwards to your application in cloud.gov, or using an alternative CDN that supports DNSSEC.
```

Comment:
Drafted some recommendations for alternate implementations in `custom-domains.md` but these are probably out of scope and would take more work to be useful to potential users.

## Additional custom domain options

While the preferred way to set up custom domains is through the cloud.gov [CDN service]({{< relref "docs/services/cdn-route.md" >}}), you can set up a custom domain without it. 74 You'll need a proxy server within your boundary or your own CDN service which terminates TLS connections:

### Proxy server

1. Obtain a TLS certificate for your preferred domain, and configure your proxy to present that certificate.
1. Point your domain to the proxy server via `A` and `AAAA` DNS records
1. Configure the proxy `ProxyPass` to forward traffic to your app `APPNAME.app.fr.cloud.gov`

### Custom CDN service

1. Obtain a TLS certificate for your preferred domain, and configure your CDN to present that certificate.
1. Point your domain to the CDN service via `CNAME` DNS record
1. Configure the CDN service `Origin` to forward traffic to your app `APPNAME.app.fr.cloud.gov`

 
***Note: need to work out general behavior for Host: header. CF Router needs to know this is for the corresponding customer app. ***

Reply:
That makes sense to me. Probably just put in a note that they can contact us if they need implementation recommendations? And we can send them these notes and work out the details with them.